recipes/geotrans/all/conanfile.py

```diff
@@ -0,0 +1,95 @@
+from conans import ConanFile, CMake, tools
+import os
+import shutil
+
+
+class GeotransConan(ConanFile):
+    name = "geotrans"
+    version = "3.8"
+    license = (
+        "NGA GEOTRANS ToS (https://earth-info.nga.mil/php/download.php?file=wgs-terms)"
+    )
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://earth-info.nga.mil/"
+    description = "MSP GEOTRANS is the NGA and DOD approved coordinate converter and datum translator."
+    topics = (
+        "geotrans",
+        "geodesic",
+        "geographic",
+        "coordinate",
+        "datum",
+        "geodetic",
+        "conversion",
+        "transformation",
+    )
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    generators = "cmake"
+    exports_sources = "CMakeLists.txt"
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(
+            **self.conan_data["sources"][self.version],
+            strip_root=True,
+            destination=self._source_subfolder,
+            filename="master.tgz"
+        )
+        shutil.copy("CMakeLists.txt", self._source_subfolder)
```

Comment:
You don't need this. That's what `exports_sources` is for. Also, avoid most of the python std lib (because Conan can actually be installed without python) and use:
https://docs.conan.io/en/latest/reference/tools.html
Whenever applicable.